codebase/superdarn/src.idl/lib/efield.1.4/efield.pro

```diff
@@ -2,30 +2,26 @@
 ; ==========
 ; Author: R.J.Barnes
 ; 
-; LICENSE AND DISCLAIMER
 ; 
-; Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
-; 
-; This file is part of the Radar Software Toolkit (RST).
+;  Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
 ; 
 ; RST is free software: you can redistribute it and/or modify
-; it under the terms of the GNU Lesser General Public License as published by
+; it under the terms of the GNU General Public License as published by
 ; the Free Software Foundation, either version 3 of the License, or
-; any later version.
+; (at your option) any later version.
 ; 
-; RST is distributed in the hope that it will be useful,
+; This program is distributed in the hope that it will be useful,
 ; but WITHOUT ANY WARRANTY; without even the implied warranty of
-; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-; GNU Lesser General Public License for more details.
-; 
-; You should have received a copy of the GNU Lesser General Public License
-; along with RST.  If not, see <http://www.gnu.org/licenses/>.
+; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+; GNU General Public License for more details.
 ; 
+; You should have received a copy of the GNU General Public License
+; along with this program. If not, see <https://www.gnu.org/licenses/>.
 ; 
+; Modifications:
 ; 
-;
-;
-; PublicFunctions
+; RST is free software: you can redistribute it and/or modify
+; it under the terms of the GNU General Public License as published by
 ; ---------------
 ;
 ; EFieldRead        
```

Comment:
Similar to above, the "Public Functions" list header appears to have been inadvertently removed from this and several other IDL library files.

Reply:
Should be able to take care of this one as well.  Thanks again for catching these!

Reply:
This is resolved with ce449d1.

Reply:
Whoops, this specific one is actually fixed with f4842ac.